tests/test_auth.py

```diff
@@ -45,6 +46,78 @@ def test_get_user_auth_token_when_file_does_not_exist(self):
         assert response == expected
 
 
+class TestValidateUserAuthTokenByProfileWithValidToken(BaseTestClass):
+    def setup(self):
+        self.token_data = "a" * 40
```

Comment:
@takitsuse rather than using this can you use random string generator. That makes code more readable and we don't need to hardcode here